1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_FizzbuzzV.2/src/index.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { NonNumberTypeError, NumberOutOfBoundsError, fizzbuzz } from "./fizzbuzz";
+
+describe('fizzbuzz', () => {
+    describe("takes numbers from 1 to 100 and outputs them as a string", () => {
+        it.each([[1, "1"], [2, "2"], [2.6, "2.6"], [4, "4"], [7, "7"], [11, "11"], [13, "13"], [17, "17"], [98, "98"]])("when given %s it returns '%s'", (input, expected) => {
+            expect(fizzbuzz(input)).toBe(expected)
+        })
+    });
+    describe('gracefully handles out-of-bound numbers below 1 and above 100', () => {
+        it.each([0, 101, -12, 102, -34, -5, -3, -9, 115, 112])("when given %s it throws an error", (input) => {
+            expect(() => fizzbuzz(input)).toThrowError(NumberOutOfBoundsError.message)
+        })
+        it('does not throw an error when given a valid input', () => {
+            expect(() => fizzbuzz(1)).not.toThrowError(NumberOutOfBoundsError.message)
+        })
+    })
+    describe('returns "Fizz" when given a multiple of 3', () => {
+        it.each([[3, "Fizz"], [6, "Fizz"], [9, "Fizz"], [12, "Fizz"], [18, "Fizz"], [21, "Fizz"], [24, "Fizz"], [27, "Fizz"], [33, "Fizz"], [36, "Fizz"], [39, "Fizz"], [42, "Fizz"], [48, "Fizz"], [51, "Fizz"], [54, "Fizz"], [57, "Fizz"], [63, "Fizz"], [66, "Fizz"], [69, "Fizz"], [72, "Fizz"], [78, "Fizz"], [81, "Fizz"], [84, "Fizz"], [87, "Fizz"], [93, "Fizz"], [96, "Fizz"], [99, "Fizz"]])("when given %s it returns '%s'", (input, expected) => {
```

Comment:
The only think add there is that this "Fizz" | "Buzz" | "FizzBuzz" Could be constants in your code used here. If let's say for some reason this values changes you won't have the trouble to update it in 1 million places. But overall this code looks really good! 
🚀